tests/test_transforms/test_math/test_limit_transform.py

```diff
@@ -0,0 +1,129 @@
+import numpy as np
+import pandas as pd
+import pytest
+
+from etna.datasets import TSDataset
+from etna.metrics import MSE
+from etna.models import ProphetModel
+from etna.pipeline import Pipeline
+from etna.transforms.math import LimitTransform
+
+
+@pytest.fixture
+def ts_check_pipeline_with_limit_transform(random_seed) -> TSDataset:
```

Comment:
Too complicated fixturs, let's separate them into the following ones
1. ts_with_exod, ts_with_transformed_target(several columns with transformed or several fixtures), ts_with_transformed_exog(several columns with transformed or several fixtures) -- will be used in test_transform
2. ts_with_quantiles, ts_with_quantiles_inverse_transformed(several columns with transformed or several fixtures) -- will be used in test_inverse_transform(in fact you can add quantiles in ts_with_exod)